packages/vuetifyjs.com/src/App.vue

```diff
@@ -45,3 +45,9 @@
     }
   }
 </script>
+
+<style>
+  .wf-loading .material-icons {
+    display: none;
```

Comment:
@MajesticPotatoe maybe change this line to `opacity` can fix the example copy icon.
```suggestion
    opacity: 0;
```

Reply:
It's probably vuetifyjs/vuetifyjs.com#353

`display: none` is needed so the icon text doesn't displace other elements.

Reply:
Tried the opacity, no go. Seems that `display: inline-flex;` on `.v-icon` is getting overridden somewhere or is being ignored.
adding 
```suggestion
.v-icon {
    display: inline-flex;
}
```
into App.vue does seem to fix it though (but this seems hacky)

Reply:
Yeah that's vuetifyjs/vuetifyjs.com#353, `.material-icons` adds `display: inline-block`

Reply:
AH! gotcha, sorry, read it too fast. Seems just need to move `import 'vuetify/dist/vuetify.css';` below the WebFontLoader in `entry-client.js`

Reply:
es6 imports are hoisted